lib/src/web/readers/configurations/network_configuration_reader.dart

```diff
@@ -17,11 +17,15 @@ class NetworkConfigurationReader extends NetworkConfiguration {
             endpoint: map['endpoint'],
             method: map['method'] == null
                 ? null
-                : Method.values.byName(map['method']));
+                : Method.values.byName(map['method']),
+            customPostPath: map['customPostPath']);
 
   void addTrackerOptions(dynamic options) {
     if (method != null) {
       options['eventMethod'] = method?.name;
     }
+    if (customPostPath != null) {
+      options['postPath'] = customPostPath;
```

Comment:
Here I think we actually need to add the leading slash to the path. Just tried it and the JS tracker is expecting the leading slash, otherwise it just joins the post path to the domain without any separation.

Reply:
You're right, I got mixed up when I was testing this yesterday 😞